github/repos.go

```diff
@@ -1122,6 +1123,14 @@ func (s *RepositoriesService) Transfer(ctx context.Context, owner, repo string,
 	r := new(Repository)
 	resp, err := s.client.Do(ctx, req, r)
 	if err != nil {
+		// Persist AcceptedError's metadata to the Repository object.
+		if aerr, ok := err.(*AcceptedError); ok {
```

Comment:
Hm, I guess transfer was never updated to handle `AcceptedError`, but probably should have since it also contains metadata. Im actually going to remove this change to keep it to be the same behavior.